src/backend/rtlsym.d

```diff
@@ -0,0 +1,179 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1994-1998 by Symantec
+ *              Copyright (c) 2000-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     backendlicense.txt
+ * Source:      $(DMDSRC backend/_rtlsym.d)
+ */
+
+module ddmd.backend.rtlsym;
+
+import ddmd.backend.cc : Symbol;
+
+enum RTLSYM
+{
+    throwd,
+    throwc,
+    throwdwarf,
+    monitor_handler,
```

Comment:
https://github.com/dlang/dmd/blob/master/src/backend/rtlsym.h

The full names are generated by macro expansion here:

https://github.com/dlang/dmd/blob/master/src/backend/cc.h#L1616

and are used in both the front end C++ source and the backend C++ source.